### ISSUE TITLE
Console text color doesn't change depending on browser theme

### ISSUE THREAD

**Issue by Xorok** [tarampampam/random-user-agent#51]
## Steps to reproduce:
 1. Use Firefox
 2. Show web console
 3. Enable dark theme in settings

## Expected behaviour:
1. Console text should use theme information and change like other console output.

## Actual behaviour:
1. Text stays black and is hard to read/has bad contrast.

---------

### System information:
**Browser version**:
```
Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:60.0) Gecko/20100101 Firefox/60.0
```

**Extension version**:
```bash
For example: 2.2.0
```

**Console screenshot**:
![Console screenshot](https://i.imgur.com/6xYQod6.png)

**Comment by neroux** [comment]
Thanks for the report. The issue derives from the hardcoded font style in [`consoleMessage`](https://github.com/tarampampam/random-user-agent/blob/master/extension/js/inject/content.js#L28).

**Comment by Xorok** [comment]
No, I don't know any API. Considering the fact that you can get/set the current browser theme as a WebExtension, and integrate new panels in the console window, this seems like a bug to me. Especially Extension-defined panels don't have any way to blend in with the dark mode. You should probably file a request/bug over at Bugzilla.

**Comment by neroux** [comment]
Thanks, it would be much appreciated if you could report that issue with Mozilla.

If there is no quick fix I presume the wisest course of action would be to fall back to the default output, without any colour customisation.

**Comment by Xorok** [comment]
The bug is here btw, but here is not a lot of activity atm. I'm not sure if there is a star/upvote system for bugs on Bugzilla?

https://bugzilla.mozilla.org/show_bug.cgi?id=1444232

**Comment by neroux** [comment]
Thanks Adrian, judging by experience that could become an endless bug :)

I guess, unless Mozilla decides to surprise us all, the best course of action would be to simply remove the styling.

**Comment by neroux** [comment]
Fixed in 2.2.3
